### ISSUE TITLE
[6.1.0+] Cannot add comments to cells from context menu

### ISSUE THREAD

**Issue by epsil** [handsontable/handsontable#5614]
### Description

It is not possible to add comments to cells by choosing `Add comment` from the context menu. The comment field disappears before one is able to type anything.

### Steps to reproduce

1.  Open the demo below to get a large table with comments and context menu enabled.
2.  Right-click on the upper-left `2017` cell and select `Add comment` from the context menu.
3.  The comment field appears briefly. However, as soon as one moves the cursor towards it, it disappears.

It seems that the comment field disappears the moment one moves the cursor over a cell. If one right-clicks on the rightmost column, so that the context menu is outside the table, then one can reach the comment field without hovering over any cells.

Specifically, the bug is triggered only if the cursor is above a cell after interacting with the context menu. The comment field disappears when moving the cursor from that cell to another cell.

In other words, this bug doesn't affect "small" tables (i.e., smaller than the context menu). The demo initializes the table with a long `data` array to ensure that the resulting table is large enough.

### Demo

http://jsfiddle.net/uhx6bL42/

### Environment

-   Handsontable version: 6.2.0
-   Browser Name and version: Firefox Quantum 63.0.3 (also tested with Google Chrome 70.0.3538.102)
-   Operating System: Microsoft Windows 10 Enterprise

**Comment by AMBudnik** [comment]
Thank you for sharing. I have replicated the issue. 
It looks like `6.1.0` is responsible for this bug. In `6.0.1` it worked fine.

**Comment by zekrix** [comment]
Any news here?
I updated Handsontable to latest and encountered same issue. Downgraded to earlier version, but has anyone come up with a proper fix?

**Comment by AMBudnik** [comment]
I hope that we will be able to focus on this in February. The Regression bugs should be fixed in the first place but know we have something super-important that we need to finish (2-3 more weeks).

**Comment by AMBudnik** [comment]
The issue should be solved with https://github.com/handsontable/handsontable/pull/5631

**Comment by AMBudnik** [comment]
Hey @epsil @zekrix 

we've just got it fixed in v `7.0.0`. Thank you for your feedback.

**Comment by ChandraAdapa10** [comment]
Hi AMBudnik,

still we are seeing this issue in nghandsontable version 5.1.0
could you please confirm is this issue solved in nghandsontable or not?

**Comment by AMBudnik** [comment]
Hi @ChandraAdapa10 

ngHandsontable hasn't been updated for a while, as we have focus on the https://github.com/handsontable/angular-handsontable, and its latest version is 5.1.1 https://github.com/handsontable/angular-handsontable/blob/master/package.json that allows you to update to any 7-8.x.x (8 excluded) version of Handsontable.

**Comment by ChandraAdapa10** [comment]
Hi AMBudnik,

thanks for your quick reply , we will update to latest 7-8.x.x version and will let you know if we see any issue.

Thanks Again

**Comment by bloodstrawberry** [comment]
There is work around for this issue.
just add ["custom add comment" at context menu](https://bloodstrawberry.tistory.com/1460). 
![img](https://github.com/handsontable/handsontable/assets/40860674/79c5b383-938f-4a28-aef4-3131721b7dad)
